hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/records/SnapshotModeOp.java

```diff
@@ -159,13 +159,23 @@ public class SnapshotModeOp extends UtilOp implements SnapshotOp {
 
     public static void main(String... args) throws IOException {
         // Helper to review the snapshot saved for a particular HapiSuite-HapiSpec combination
-        final var snapshotFileMeta = new SnapshotFileMeta("CryptoTransfer", "okToRepeatSerialNumbersInBurnList");
-        final var snapshot = loadSnapshotFor(PROJECT_ROOT_SNAPSHOT_RESOURCES_LOC, snapshotFileMeta);
+        final var snapshotFileMeta = new SnapshotFileMeta("ContractCall", "MultipleSelfDestructsAreSafe");
```

Comment:
I get that this is just a CLI utility for helping diagnose failing BDDs, but I don't get why/when you'd change the `SnapshotFileMeta` parameters?

Reply:
Ah interestingly this _is_ important, as a `BufferedWriter` will not necessarily flush its contents to the file without it!

Reply:
I did get _that_ ... what I _didn't_ get was why you'd change the arguments to `SnapshotFileMeta` but that was because I stupidly _didn't look at the constructor signature_ which tells all.